89.md

```diff
@@ -0,0 +1,81 @@
+NIP-89
+======
+
+Bitcoin Transaction Package Relay
+-------------------------
+
+`draft` `optional` `author:benthecarman`
+
+A special event with kind `28333`, that contains encoded bitcoin transactions.
+The bitcoin transactions are encoded as a hex string, and are contained in a `transactions` tag.
+
+The network magic is contained in the `tags` field. This is to identify the network that the transactions are for.
+The common ones are:
+
+- mainnet: f9beb4d9
+- testnet3: 0b110907
+- signet (default): 0a03cf40
```

Comment:
I think since money is involved, I think it's good to be explicit when using mainnet.
Don't want someone send something on mainnet by accident. However, even addresses contain net information:

> Invoice addresses on the Bitcoin Testnet are generated with a different prefix. See [List of address prefixes](https://en.bitcoin.it/wiki/List_of_address_prefixes) and [Testnet](https://en.bitcoin.it/wiki/Testnet) for more details.

-- https://en.bitcoin.it/wiki/Invoice_address#Testnet

But I think for signet and regtest this is not the case? :thinking:

Reply:
No objection to being explicit, but maybe then always be explicit? Signet doesn't look like such a natural choice for a default to me.